interfaces.go

```diff
@@ -223,6 +223,9 @@ type Machine interface {
 
 	// BlockDevices returns all the physical and virtual block devices on the machine.
 	BlockDevices() []BlockDevice
+	// PhysicalBlockDevice returns the physical block device for the machine
```

Comment:
s/PhysicalBlockDevice/BlockDevice/

and probably not "the physical block device"